cypress.hub.config.ts

```diff
@@ -6,6 +6,13 @@ import { baseConfig } from './cypress.base.config';
 baseConfig.e2e!.specPattern = 'cypress/e2e/hub/**/*.cy.ts';
 baseConfig.e2e!.baseUrl = 'http://localhost:4102';
 baseConfig.component!.specPattern = 'frontend/hub/**/*.cy.{js,jsx,ts,tsx}';
+baseConfig.env = {
```

Comment:
These should be using the environment variables.

Reply:
SEE: https://github.com/ansible/ansible-ui#environment-variables

I'd prefer to have that as the standard way to setup connecting to a server for the products.
Having "password" in code will cause security audits of the code to not be happy.